librz/bin/format/pe/dotnet.c

```diff
@@ -57,13 +57,19 @@ static ut32 clr_max_rows(int count, ...) {
 
 static int read_image_metadata_strings(RzBuffer *b, ut32 stream_start, ut32 stream_size, Pe_image_clr *clr) {
 	RzPVector /* char[] */ *strings = rz_pvector_new(free);
+	if (!strings)
```

Comment:
from the coding prospective, please use the curly brackets also for small if/else/while/for, etc..